### PR TITLE
Implement TaskServlet::doGet to list tasks lexically.

### PR DESCRIPTION
**Pull request by aaanders**
Tasks are logged on startup but aren't easily accessible otherwise. Implement `TaskServlet::doGet` to print tasks lexically by name.

Example:
```
$ curl http://localhost:8081/tasks
echo
gc
log-level
```
TaskServlet is mapped on "/tasks/*" and any GET request not matching root 
will yield 404 or 405.

404: Task name is unknown.  
405: Task name is registered but must be invoked with POST.

### CODE REVIEW

File: dropwizard-servlets/src/main/java/io/dropwizard/servlets/tasks/TaskServlet.java [hunk 1]
```diff
@@ -91,6 +95,25 @@ public void add(Task task) {
     }
 
     @Override
+    protected void doGet(HttpServletRequest req,
+                         HttpServletResponse resp) throws ServletException, IOException {
+        if (Strings.isNullOrEmpty(req.getPathInfo())) {
+            try (final PrintWriter output = resp.getWriter()) {
+                resp.setContentType(MediaType.PLAIN_TEXT_UTF_8.toString());
+                final ArrayList<Task> tasks = new ArrayList<>(getTasks());
```
Comment (joschi) [inline dropwizard-servlets/src/main/java/io/dropwizard/servlets/tasks/TaskServlet.java]: This (and the following lines) could be replaced by:
```java
getTasks().stream()
    .map(Task::getName)
    .sorted()
    .forEach(output::println);
```

File: dropwizard-servlets/src/main/java/io/dropwizard/servlets/tasks/TaskServlet.java [hunk 2]
```diff
@@ -7,9 +7,13 @@
 import com.codahale.metrics.annotation.Metered;
 import com.codahale.metrics.annotation.Timed;
 import com.google.common.base.Charsets;
+import com.google.common.base.Strings;
 import com.google.common.collect.ImmutableMultimap;
 import com.google.common.io.CharStreams;
 import com.google.common.net.MediaType;
+import java.util.ArrayList;
```
Comment (joschi) [inline dropwizard-servlets/src/main/java/io/dropwizard/servlets/tasks/TaskServlet.java]: Please remove the unused imports. Then we're good to go.

File: dropwizard-servlets/src/main/java/io/dropwizard/servlets/tasks/TaskServlet.java (hunk 1 above)
Comment (aaanders) [inline dropwizard-servlets/src/main/java/io/dropwizard/servlets/tasks/TaskServlet.java]: Nice, thanks @joschi, nice and clean. (Grepping for `stream()` made me think it was a bad idea to use it :D )

File: dropwizard-servlets/src/main/java/io/dropwizard/servlets/tasks/TaskServlet.java (hunk 2 above)
Comment (aaanders) [inline dropwizard-servlets/src/main/java/io/dropwizard/servlets/tasks/TaskServlet.java]: Fixed

**Comment by aaanders** [comment]
Thanks for reviewing @joschi

**Pull request merged**